custom/basic-call/pages/not-found.js

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
 import MessageCard from '@custom/shared/components/MessageCard';
+import { useRouter } from 'next/router';
 
 export default function RoomNotFound() {
+  const router = useRouter();
+
   return (
     <div className="not-found">
-      <MessageCard error header="Room not found">
+      <MessageCard error header="Room not found" onBack={() => router.push('/')}>
```

Comment:
this anonymous`onBack` function can be a named function (I've given this feedback on other PRs, please use it as a general request for React demo code.)

`const returnToHomePage = () => router.push('/')`
`onBack={returnToHomePage}`